include/circt/Dialect/Verif/VerifVisitors.h

```diff
@@ -21,9 +21,11 @@ class Visitor {
   ResultType dispatchVerifVisitor(Operation *op, ExtraArgs... args) {
     auto *thisCast = static_cast<ConcreteType *>(this);
     return TypeSwitch<Operation *, ResultType>(op)
-        .template Case<AssertOp, AssumeOp, CoverOp>([&](auto op) -> ResultType {
-          return thisCast->visitVerif(op, args...);
-        })
+        .template Case<AssertOp, AssumeOp, CoverOp, ClockedAssertOp,
+                       ClockedAssumeOp, ClockedCoverOp>(
+            [&](auto op) -> ResultType {
+              return thisCast->visitVerif(op, args...);
+            })
```

Comment:
I think you also need corresponding `HANDLE(ClockedAssertOp, Unhandled);` lines a bit further down in this file.